benchmarks/src/main/java/org/questdb/PongMain.java

```diff
@@ -53,7 +67,7 @@ public static void main(String[] args) {
         // queue processor job
         workerPool.assign(workerId -> dispatcher.processIOQueue(processor));
         // lets go!
-        workerPool.start();
+        workerPoolManager.startAll(LOG);
```

Comment:
this was optional, perhaps unnecessary for this example

Reply:
done